Updating landscape section (#99)
CAKE-DRI/CAKE-DRI.github.io: _pages/landscape/add_project.md

---
title: Add details about a new CAKE-related project
layout: title-page
permalink: /landscape/add_project
---

We've got various examples of projects within the DRI-space in this section of the website, but we want more!

We have two ways that you can add details of your new or existing project showcasing any CAKE-related work:

 - <a href="https://github.com/CAKE-DRI/cake.github.io" target="_blank">Submit a pull request to the Github repository for our website</a>, adding an appropriate md file into one of the `collections/_landscape-*` directories (please feel free to ask which the most appropriate section would be for your project if you're not sure!)

>  - Use the form below. Note that if your project has an image to be displayed with the listing, this will need to be emailed separately to <cake@mjiscmail.ac.uk>, or you can submit a Github pull request to add the image to the `assets/images/project-images/` folder in our website repo: <https://github.com/CAKE-DRI/CAKE-DRI.github.io>.

Reviewer: ```suggestion
 - Use the form below. Note that if your project has an image to be displayed with the listing, this will need to be emailed separately to <cake@jiscmail.ac.uk>, or by adding an image to `assets/images/project-images/` and submitting a pull request.
```
